interpreter/Makefile

```
@@ -91,7 +91,17 @@ install:
 opam-release/%:
 	git tag opam-$*
 	git push --tags
-	opam publish
+	rm -f opam-$*.zip
+	wget https://github.com/WebAssembly/spec/archive/opam-$*.zip
+	cp wasm.opam opam
+	echo 'url {' >> opam
+	echo '  src: "https://github.com/WebAssembly/spec/archive/opam-@VERSION.zip"' >> opam
+	echo '  checksum: "md5=@MD5"' >> opam
+	echo '}' >> opam
+	sed -i ".tmp" s/@VERSION/$*/g opam
+	sed -i ".tmp" s/@MD5/`md5 -q opam-$*.zip`/g opam
+	rm opam.tmp opam-$*.zip
```

If we are producing this via echo, we don't need sed but can splice in the right strings right here:
```suggestion
	echo "  src: \"https://github.com/WebAssembly/spec/archive/opam-$*.zip\"" >> opam
	echo "  checksum: \"md5=`md5 -q opam-$*.zip`\"" >> opam
	echo "}" >> opam
        rm opam-$*.zip
```

---

Indeed. Also the `sed -i ".tmp" ...` part is not working on linux, this is a macOS thing I guess. To be portable you'd do something like `sed -i.tmp ...`.